off-chain/package.json

```diff
@@ -7,6 +7,7 @@
     "test": "test"
   },
   "scripts": {
+    "blockfrost-test": "source ./test/blockfrost.env && spago run --main Scaffold.Test.Blockfrost",
     "test": "spago run --main Test.Seath.Main",
     "e2e-serve": "make e2e-serve",
```

Comment:
we probably can delete all the e2e tests from ctl, at least this part won't work now that `makefile` is deleted.